client/main/newRoom-item.tsx

```diff
@@ -0,0 +1,40 @@
+import useRoomSizeConvert from 'hooks/useRoomSizeConvert';
+
+interface NewRoomItemProps {
+    latitude: number;
+    longitude: number;
+    roomID: string;
+    roomType: string;
+    dealType: string;
+    priceUnit: string;
+    deposit: number;
+    price: number;
+    floor: string;
+    struct: string;
+    roomSize: number;
+    image: number;
+    recommend: number;
+    address: string;
+}
+
+function NewRoomItem(data: Object<NewRoomItemProps>) {
```

Comment:
```suggestion
function NewRoomItem(data: Object<NewRoomItemProps>) {
```function NewRoomItem({
  latitude,
  longitude, 
  roomID, 
  roomType, 
  dealType, 
  priceUnit, 
  deposit, 
  price, 
  floor,
  struct, 
  roomSize, 
  image, 
  recommend, 
  address
}: NewRoomItemProps) {